Translations/fr.po

```diff
@@ -10976,7 +10976,7 @@ msgid ""
 " \n"
 "Perhaps I can tell you more if we speak again. Good luck."
 msgstr ""
-"Dieu merci, vous êtes de retour !\n"
+"Dieu merci, vous êtes de retour !\n"
```

Comment:
Any idea what the change here is?

Reply:
Hexdump says it used to be a non-breaking space, and it got replaced with a space. Come to think of it, should there be a space there at all?

Reply:
I don't speak French, but maybe @Phalcon22 knows :)

Reply:
Apparently it's correct. https://www.quora.com/Why-do-French-speakers-place-a-space-before-exclamation-points

Reply:
I confirm it is correct.

Reply:
I confirm it is correct.

Reply:
Ok, if you can revert that to be NBS then I think we can merge this :)

Reply:
Done.